src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Services/PreviewTokenService.cs

```diff
@@ -85,7 +85,8 @@ private async Task<bool> HasActivePreviewToken(
             .Where(pt => pt.Id == previewTokenId)
             .Where(pt => pt.DataSetVersionId == dataSetVersionId)
             .Where(pt => pt.DataSetVersion.Status == DataSetVersionStatus.Draft)
-            .Where(pt => pt.Expiry > timestamp)
+            .Where(pt => (pt.Activates == null || pt.Activates <= timestamp) 
+                         && pt.Expiry > timestamp)
```

Comment:
See my other comment about possibly making `Activates` non-nullable, in which case this can be simplified.